tutorials/tagger/pos_allennlp_bare.py

```diff
@@ -0,0 +1,130 @@
+# from https://pytorch.org/tutorials/beginner/nlp/sequence_models_tutorial.html
+# pylint: disable=invalid-name,arguments-differ
+from typing import Iterator, List
+
+import torch
+import torch.optim as optim
+
+from allennlp.data import Instance
+from allennlp.data.iterators import BasicIterator
+from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.fields import TextField, SequenceLabelField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import Token
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models import Model
+from allennlp.modules.feedforward import FeedForward
+from allennlp.modules.text_field_embedders import TextFieldEmbedder, BasicTextFieldEmbedder
+from allennlp.modules.token_embedders import Embedding
+from allennlp.modules.seq2seq_encoders import Seq2SeqEncoder, PytorchSeq2SeqWrapper
+from allennlp.nn.util import get_text_field_mask, sequence_cross_entropy_with_logits
```

Comment:
that's a lot of imports!

Reply:
You get around this in the plain pytorch version by importing `nn` and doing `nn.X`.  You could do the same here, if you wanted to.